core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -169,9 +170,9 @@ private static Properties readProperties(URL url) {
         try (InputStream inputStream = url.openStream()) {
             properties.load(inputStream);
         } catch (FileNotFoundException e) {
-            log.trace("Testcontainers config override was found on {} but the file was not found", url, e);
+            log.warn("Testcontainers config override was found on {} but the file was not found", ExceptionUtils.getRootCauseMessage(e));
```

Comment:
I could be reading it wrong, but won't this insert the exception message right into the middle of the log string in a very strange way?

e.g. if the exception message is `something went wrong` then we're going to end up logging:

`Testcontainers config override was found on something went wrong but the file was not found`

I think we want:
```suggestion
            log.warn("Testcontainers config override was found on {} but the file was not found. Exception message: {}", url, ExceptionUtils.getRootCauseMessage(e));
```

and similar for the `IOException` case...

WDYT?

Reply:
Thanks @rnorth . Fixing.